src/hotkey/command_executor.cpp

```diff
@@ -33,6 +33,7 @@
 #include "show_dialog.hpp"
 #include "../resources.hpp"
 #include "../playmp_controller.hpp"
+#include "sdl/input.hpp" // get_mouse_state
```

Comment:
Is that standard style in wesnoth? (I honestly have no clue, but I haven't seen it before)

Reply:
if it isn't already standard style, it ought to be, IMO

Reply:
I have literally no idea what this comment is referring to?

Reply:
@CelticMinstrel I think it tries to explain what exactly of `sdl/input.hpp` will be used in this file. However, since it is not compiler-enforced anyway, I'd rather not have those. But I don't know wesnoth style sufficiently to say, thus just raising as a question.

Reply:
Some files have these, some files don't. I prefer to include them but can tidy it up so it follows the style of the file it's in, if desired.

The other files that do this write it as "for get_mouse_state" etc, so i could also change it to that to make it more clear.

Reply:
Oh, that. I usually only add that if it's not obvious – that is, you're including a file for some auxiliary thing it contains rather than the main thing in that file.